controllers/index.js

```diff
@@ -0,0 +1,5 @@
+const user = require("./user.controller");
+
+module.exports = {
+  user,
+};
```

Comment:
you can skip this file by adding controller itself in the route file.
same as you are importing services into the controller

Reply:
okay.